source/tutorial/autocomplete-component.md

```diff
@@ -510,21 +510,34 @@ Open our existing acceptance test, `tests/acceptance/list-rentals-test.js`, and
 
 
 ```/tests/acceptance/list-rentals-test.js
-test('should filter the list of rentals by city.', function (assert) {
-  visit('/');
-  fillIn('.list-filter input', 'Seattle');
-  keyEvent('.list-filter input', 'keyup', 69);
-  andThen(function() {
-    assert.equal(find('.listing').length, 1, 'should show 1 listing');
-    assert.equal(find('.listing .location:contains("Seattle")').length, 1, 'should contain 1 listing with location Seattle');
-  });
+test('should filter the list of rentals by city', async function(assert) {
+  await visit('/');
+  await fillIn('.list-filter input', 'seattle');
+  await triggerKeyEvent('.list-filter input', 'keyup', 69);
+  assert.equal(findAll('.results .listing').length, 1, 'should display 1 listing');
```

Comment:
assert.ok(find(...)) should be sufficient here if you only want to check for the existance of something

Reply:
also find/findAll mainly exist to ease the migration from the old testing APIs. new code, according to @rwjblue, should use this.element.querySelector() (or qunit-dom one the RFC is merged)

Reply:
won't qunit-dom back find when that happens?  According to https://github.com/emberjs/ember-test-helpers/blob/master/API.md#find that might be happening already

Reply:
I don't understand the question. qunit-dom is not using `find()` under the hood, but `document.querySelector()` instead.

Reply:
sorry.  wasn't clear.  ember-test-helpers find is using querySelector

Reply:
I see what you are saying.  querySelecter is done at root with find

Reply:
Is this.element documented somewhere?

Reply:
does this.element return the root in an acceptance test anyways (sorry for the multiple comments, my thinking is spaced out while I work lol)